global_config.json

```diff
@@ -1,5 +1,6 @@
 {
-  "sdkVersion": "1.11", // The version of the Answers SDK to use
+  "sdkVersion": "develop", // The version of the Answers SDK to use
```

Comment:
make sure you update this to the latest sdk version!

Reply:
yes, will do once the sdk is cut!